src/util/views/records.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+import logging
+from contextlib import contextmanager
+
+# python3 shims
+try:
+    unicode  # noqa: B018
+except NameError:
+    unicode = str
+
+import lxml
+from psycopg2.extras import Json
+
+try:
+    from odoo.tools.translate import xml_translate
+except ImportError:
+    xml_translate = lambda callback, value: value
+
+from ..pg import column_exists, column_type
+from ..report import add_to_migration_reports
+
+_logger = logging.getLogger(__name__)
+
```

Comment:
__all__ = ["remove_view", "edit_view", "add_view"]

Reply:
`__all__` for the record.